src/assets/icons/ShareFilled.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we should have only one icon, and it should be possible to "fill" using css.
If it doesn't work with the `Share.svg` icon then we need to ask the designers for a better asset.
But this file should be removed.